build.gradle

```diff
@@ -217,6 +229,10 @@ checkScoverage {
     minimumRate = 0.92
 }
 
+coveralls {
+    coberturaReportPath = "${buildDir}/reports/scoverage/cobertura.xml"
```

Comment:
by default this plugin expects the report to be in `build/reports/cobertura/coverage.xml`

i used the extension property here 
https://github.com/kt3k/coveralls-gradle-plugin/blob/11be24eb68dbc8c9348f62b1704529b05811de3c/src/main/groovy/org/kt3k/gradle/plugin/CoverallsPluginExtension.groovy#L23